rust/Cargo.toml

```diff
@@ -71,6 +71,7 @@ async-trait = "0.1"
 # rust-dataframe = {version = "0.*", optional = true }
 
 [features]
+# default = ["datafusion-ext"]
```

Comment:
one minor ask, i think this should be removed?

Reply:
done